src/client/app/components/TooltipHelpComponent.tsx

```diff
@@ -27,51 +27,54 @@ export default function TooltipHelpComponent(props: TooltipHelpProps) {
 
 
 	const version = useAppSelector(selectOEDVersion);
+	// The web pages, including help/documentation use _ not . in version.
+	//  It also uses V not v and has help at the start.
+	const helpVersion = 'help' + version.replaceAll('.', '_').replace('v', 'V');
 	const baseHelpUrl = useAppSelector(selectBaseHelpUrl);
-	const helpUrl = baseHelpUrl + version;
+	const helpUrl = baseHelpUrl + helpVersion;
```

Comment:
Works As Expected, refer to #1323 for minor suggestions in selector usage if desired, otherwise good to go.